cli/pkg/telemetry/telemetry.go

```diff
@@ -69,7 +70,12 @@ func (t *Telemetry) WithUserID(userID string) {
 }
 
 func (t *Telemetry) Emit(action Action) {
-	t.emitBehaviourEvent(string(action), "cli", "terminal", "terminal")
+	t.emitBehaviourEvent(string(action), "cli", "terminal", "terminal", nil)
+}
+
+func (t *Telemetry) EmitStartEvent(sourceDrivers []string, olapDriver string) {
+	payload := map[string]any{"source_driver": sourceDrivers, "olap_driver": olapDriver}
```

Comment:
The analyzed connectors actually not necessarily just for sources, e.g. if we add support for exporting to a connector, it would also show up here. So would recommend renaming `sourceDrivers` to just `connectors`. 

And maybe just to align terminology, also consider changing `olap_driver` to `olap_connector`.

Reply:
renamed